rest/tests/test_rest.py

```diff
@@ -0,0 +1,471 @@
+"""
+Unit tests for REST api
+"""
+
+from __future__ import unicode_literals
+
+import json
+from django.db.models.deletion import ProtectedError
+from django.test import Client
+
+from learningresources.tests.base import LoreTestCase
+from taxonomy.api import create_vocabulary
+from taxonomy.models import Vocabulary
+from rest.serializers import VocabularySerializer
+
+HTTP_OK = 200
+HTTP_CREATED = 201
+HTTP_NO_CONTENT = 204
+HTTP_BAD = 400
+HTTP_UNAUTHORIZED = 403
+HTTP_NOT_FOUND = 404
+HTTP_METHOD_NOT_ALLOWED = 405
+
+def as_json(resp):
+    """Get JSON from response"""
+    return json.loads(resp.content.decode('utf-8'))
+
+class TestRest(LoreTestCase):
+    """
+    REST test
+    """
+
+    def test_repositories(self):
+        """
+        Test for Repository
+        """
+        resp = self.client.get('/api/v1/repositories/')
+        self.assertEqual(HTTP_OK, resp.status_code)
```

Comment:
in your tests you are doing this a lot: probably it makes sense to write a helper function that checks the status code (and maybe returns the resp or directly the JSON)